server/lib/payments.js

```diff
@@ -451,7 +451,7 @@ const sendOrderProcessingEmail = async order => {
     user: user.info,
     collective: collective.info,
     host: host.info,
-    fromCollective: fromCollective.minimal,
```

Comment:
I don't see the point of this change.

Reply:
need to pass the `isIncognito` to the email template.